src/FinchNotation/FinchNotation.jl

```diff
@@ -34,10 +34,13 @@ module FinchNotation
 
     export getval, getname
 
-    export overwrite, initwrite, MakeDimension, mkdim
+export overwrite, initwrite, MakeDimension, mkdim
+
+export parallelAnalysis, ParallelAnalysisResults
 
     include("nodes.jl")
     include("instances.jl")
     include("protocols.jl")
-    include("syntax.jl")
-end
\ No newline at end of file
+include("syntax.jl")
+include("analysis.jl")
```

Comment:
syntax and analysis should both live in the transforms subdirectory. FinchNotation is just a module to define the AST

Reply:
There already is a syntax.jl in transforms. I'd argue analysis is not a transform, but it also doesn't matter so I will move that.